web-ui/src/main/resources/catalog/components/admin/uiconfig/partials/projectionSwitcher.html

```diff
@@ -24,39 +24,43 @@
 				<p class="help-block" data-translate="">ui-definition-help</p>
 			</div>
 			<div class="input-group">
-				<span class="input-group-addon" data-translate="">ui-extent</span> <input
-					type="text" class="form-control" gn-json-edit="opt.extent" />
+				<span class="input-group-addon" data-translate="">ui-extent</span>
+        <input
+					type="text" class="form-control"
+          gn-json-edit="opt.extent"
+          readonly="readonly"/>
 			</div>
 			<div class="input-group">
 				<span class="input-group-addon" data-translate="">ui-minx</span> <input
-					type="number" class="form-control" data-ng-model="opt.extent[0]" />
+					type="number" step="any" class="form-control" data-ng-model="opt.extent[0]" />
 				<span class="input-group-addon" data-translate="">ui-miny</span> <input
-					type="number" class="form-control" data-ng-model="opt.extent[1]" />
+					type="number" step="any" class="form-control" data-ng-model="opt.extent[1]" />
 				<span class="input-group-addon" data-translate="">ui-maxx</span> <input
-					type="number" class="form-control" data-ng-model="opt.extent[2]" />
+					type="number" step="any" class="form-control" data-ng-model="opt.extent[2]" />
 				<span class="input-group-addon" data-translate="">ui-maxy</span> <input
-					type="number" class="form-control" data-ng-model="opt.extent[3]" />
+					type="number" step="any" class="form-control" data-ng-model="opt.extent[3]" />
 			</div>
 			<div class="input-group">
 				<p class="help-block" data-translate="">ui-extent-help</p>
 			</div>
 			<div class="input-group">
 				<span class="input-group-addon" data-translate="">ui-worldExtent</span>
 				<input type="text" class="form-control"
-					gn-json-edit="opt.worldExtent" />
+               gn-json-edit="opt.worldExtent"
+               readonly="readonly"/>
```

Comment:
If this field is read only you probably don't need to use `gn-json-edit`. Something like `ng-value="JSON.stringify(opt.worldExtent)"` might work and you won't have to deal with two-way data binding.

Actually, now that I think of it, is this field really needed? Just giving my 2c, I let you judge what's best.

Reply:
To me I would remove the field with the extent from the form and would only keep the individual coordinates.

Reply:
I've removed the UI field for `extent` and `worldExtent`.